MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -440,6 +440,10 @@
     return stgMode;
 }
 
+bool MVKImage::isMemoryHostCoherent() {
+    return (getMTLStorageMode() == MTLStorageModeShared);
```

Comment:
The call to `getMTLStorageMode()` will crash if `_deviceMemory` is not yet set for the image.

Can you either add a test for `_deviceMemory` in this method, similar to the overridden `MTLResource::isMemoryHostCoherent()`...or perhaps better...fix `MVKImage::getMTLStorageMode()` to handle a missing `_deviceMemory` (maybe defaulting to `MTLStorageModePrivate` if no `_deviceMemory`?

Reply:
no problem!